src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1090,7 +1124,7 @@ private static bool IsTypeParameterDisallowingAnnotation(TypeSymbol typeOpt)
         public override BoundNode VisitLocal(BoundLocal node)
         {
             var local = node.LocalSymbol;
-            int slot = GetOrCreateSlot(local);
+            int slot = GetOrCreateSlot(local, createIfMissing: false);
```

Comment:
>, createIfMissing: false [](start = 44, length = 24)

Does this matter? Consider removing for simplicity.

Reply:
This shouldn't matter (since a slot for the local should have been created already), but it seems reasonable to pass `false` explicitly so the intent is clear that a slot should not be created here.

---
In reply to: [246872573](https://github.com/dotnet/roslyn/pull/32304#discussion_r246872573) [](ancestors = 246872573)